### DISCUSSION TITLE
What if I want to obtain and use another ssl certificate instead of using Let's Encrypt?

### DISCUSSION THREAD

**Discussion by nokemono** [tiangolo/uwsgi-nginx-flask-docker#356] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
none
```

### Description

What if I want to obtain and use another ssl certificate instead of using Let's Encrypt?

### Operating System

Linux

### Operating System Details

none

### Python Version

3.7

**Comment by tiangolo** [comment, marked as answer]
Yep, in any case the HTTPS certificate and handling would be out of scope for what is included in this project.
